charts/flyte-core/values.yaml

```diff
@@ -200,6 +206,8 @@ flytepropeller:
   tolerations: []
   # -- affinity for Flytepropeller deployment
   affinity: {}
+  # Metric port used for collecting metric
+  metricPort: 10254
```

Comment:
You will have to put this in the config maps to take effect... it's not enough to list them here and in the container Ports...
e.g.: https://github.com/flyteorg/flytepropeller/blob/6a14e7fbffe89786fb1d8cde22715f93c2f3aff5/pkg/controller/config/config.go#L101